#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_perception'
try:
    import imp
    imp.find_module(PACKAGE)
    from dynamic_reconfigure.parameter_generator_catkin import *;
except:
    import roslib; roslib.load_manifest(PACKAGE)
    from dynamic_reconfigure.parameter_generator import *;

from math import pi

gen = ParameterGenerator ()

gen.add("scaling_factor", double_t, 0,"sliding window scaling factor in each stack",
        -0.05, -0.90, 0.9)
gen.add("stack_size", int_t, 0, "number of image stack in pyramid", 4, 1, 15)
gen.add("sliding_window_increment", int_t, 0, "number of pixels to increment the window",
        8, 1, 32)

exit (gen.generate (PACKAGE, "jsk_perception", "TrainedObjectDetector"))